<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>263</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="dgvDadosProgramados">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>511</width>
      <height>201</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <column>
     <property name="text">
      <string>id</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>titulo</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>nota</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>data</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>status</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="btnNovo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>sans-serif</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#btnNovo{
color:#484848;
font-family:sans-serif;
}</string>
    </property>
    <property name="text">
     <string>Concluida</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
